tests/fake_api.py

```diff
@@ -323,6 +323,7 @@ def osbs(openshift):
 koji_root = http://koji.example.com/kojiroot
 koji_hub = http://koji.example.com/kojihub
 use_auth = false
+can_orchestrate = true
```

Comment:
I was going to suggest adding this to osbs106 too but it seems that's pretty stale.